releasenotes/notes/sampler_as_positional_arg_for_qnspsa-2efca170947f3cd7.yaml

```
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    The list of arguments for `QNSPSA.get_fidelity()` has been updated to
+    include only the `ansatz` and the `sampler`, and removing previous
+    wildcard option. From now on, the sampler option must be explicitly
+    specified, as a positional or keyword argument, and it cannot be `None`.
```

```suggestion
    The list of arguments for `QNSPSA.get_fidelity()` has been updated to
    remove the keyword enforcement for `sampler` so also allowing it now
    as positional. Additionally the `sampler` must be explicitly specified, as a
    valid object, as it no longer supports `None` nor defaults to that.
```

What you call a "wildcard" option I am guessing refers to the '*' which in that context is a keyword separator meaning any arguments following that must be supplied as keyword args. (There is a also a / that is possible in signature where
args preceeding that can only be positional and supplying them in keyword form will cause an error - I think in algs
only the * is ever used)

Anyway I updated the text that hopefully reflects the change a bit better.